src/components/Banner.tsx

```diff
@@ -0,0 +1,33 @@
+/** @jsxImportSource @emotion/react */
+import { css, useTheme } from '@emotion/react';
+
+const bannerStyle = (theme: any) => css`
```

Comment:
```suggestion
const bannerStyle = (theme: ThemeType) => css`
```

theme.ts 파일에 타입 잘 선언해놓으셨던데.. 그걸 활용해보면 좋을 거 같습니다!

나머지 `:any` 사용하신 부분들도 ThemeType으로 바꿔주세요.
any를 안 쓰려고 노력할수록 TypeScript 실력도 올라가고, 동료들도 좋아합니다. 🥹

Reply:
피드백 주신 부분 반영했습니다!
theme: any → ThemeType으로 수정하고, 관련된 함수들 모두 타입 지정 완료했습니다.
앞으로도 any 줄이는 습관 계속 가져가겠습니다 😊